backend/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -707,8 +712,19 @@ public static Predicate getValuePredicate(
         // builder.literal(10),builder.literal(10))),
         //            operator, value.getNumberValue());
         if (ModelDBHibernateUtil.rDBDialect.equals(ModelDBConstants.POSTGRES_DB_DIALECT)) {
-          return getOperatorPredicate(
-              builder, valueExpression.as(Double.class), operator, value.getNumberValue());
+          if (stringColumn) {
+
+            return getOperatorPredicate(
+                builder,
+                builder
+                    .trim(Trimspec.BOTH, Character.valueOf('"'), valueExpression)
```

Comment:
tests/ db call broke on OSS when we add "2" in value and the predicate came with a numeric value